tvb_documentation/demos/Demos.rst

```diff
@@ -324,6 +326,22 @@ Interact with TVB Framework
 
       `Generate TimeSeries for import in Web GUI`_
 
+Interact with TVB REST Client API
+---------------------------
```

Comment:
Minor, but rendering won't work if the underlining part is not covering the entire title :-)